src/components/rightSidePanel/parameters/TabNodes.vue

```
@@ -30,6 +32,38 @@ const searchedWidgetsSectionDataList = shallowRef<NodeWidgetsListList>(
   widgetsSectionDataList.value
 )
 const isSearching = ref(false)
+
+const collapseMap = reactive<Record<string, boolean>>({})
+
+watch(
+  () => workflowStore.activeWorkflow?.path,
+  () => {
+    Object.keys(collapseMap).forEach((key) => delete collapseMap[key])
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)